fdbserver/ClusterController.actor.cpp

```diff
@@ -1719,6 +1736,11 @@ class ClusterControllerData {
 				} else if (regions[0].dcId == clusterControllerDcId.get()) {
 					return reply.get();
 				}
+				TraceEvent(SevWarn, "CCRecruitmentFailed", id)
+				    .detail("Reason", "Recruited Txn system and CC are in different DCs")
+				    .detail("CCDcID", clusterControllerDcId.get())
+				    .detail("RecruitedTxnSystemDcID", regions[0].dcId)
+				    .detail("Action", "CC tries to recruit in its DC");
```

Comment:
It's possible you don't need this line, as the `Reason` field gives roughly the same info.

```suggestion
				    .detail("Action", "CC tries to recruit in its DC");
```